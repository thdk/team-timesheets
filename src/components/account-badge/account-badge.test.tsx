import React from "react";
import { render } from "@testing-library/react";
import { AccountBadge } from "./";

describe("AccountBadge", () => {
    it("should render", () => {
        const { asFragment } = render(
            <AccountBadge
                name="Foobar"
                email="foobar@example.com"
            />
        );

        expect(asFragment()).toMatchSnapshot();
    });
});