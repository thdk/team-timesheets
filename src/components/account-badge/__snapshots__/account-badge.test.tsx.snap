// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountBadge should render 1`] = `
<DocumentFragment>
  <div
    class="account-badge"
  >
    <div
      class="account-badge__content"
    >
      <div
        class="account-badge__image"
      >
        <img
          src="https://www.gravatar.com/avatar/0d4907cea9d97688aa7a5e722d742f71?d=mp&s=40"
        />
      </div>
      <div
        class="account-badge__name"
      >
        Foobar
      </div>
    </div>
  </div>
</DocumentFragment>
`;
